packages/webapp/src/views/common/applications.html

```diff
@@ -53,6 +53,9 @@ <h1 class="govuk-heading-xl">{{ pageHeading }}</h1>
               {% endfor %}
             </tbody>
           </table>
+        {%if showAmendmentGuidanceLink %}
```

Comment:
For consistency, should this be `{% if` as this is how we've written it elsewhere?

```suggestion
        {% if showAmendmentGuidanceLink %}
```